guide/spanish/react/what-are-react-props/index.md

```diff
@@ -1,7 +1,8 @@
----
-title: React TypeChecking with PropTypes
-localeTitle: Reaccionar TypeChecking con PropTypes
----
## Reaccionar PropTypes
+---
+title: React TypeChecking with PropTypes
+localeTitle: Reaccionar TypeChecking con PropTypes
+---
+## Reaccionar PropTypes
```

Comment:
Still not sure what is going on here.. but the result looks legit! Some hidden character..

Reply:
@lipis Getting close to merging this one.  What is the issue you are concerned about here?

Reply:
React shouldn't be translated. TypeChecking is bad already in the original.

Reply:
@lynxlynxlynx If you look at the changes, it does look like what shows above.  I think this is a non-issue.